Sources/MongoSwift/MongoClient.swift

```diff
@@ -39,6 +39,11 @@ public struct ClientOptions: CodingStrategyProvider, Decodable {
      */
     public var threadPoolSize: Int?
 
+    /// Whether or not to require TLS for connections to the server
```

Comment:
nits: 1) maybe helpful to state default is to not require TLS? 2) end sentence with period

Reply:
done and done